osu.Game/Online/Leaderboards/UpdateableRank.cs

```diff
@@ -7,23 +7,47 @@
 
 namespace osu.Game.Online.Leaderboards
 {
-    public class UpdateableRank : ModelBackedDrawable<ScoreRank>
+    public class UpdateableRank : ModelBackedDrawable<ScoreRank?>
     {
-        public ScoreRank Rank
+        public ScoreRank? Rank
         {
             get => Model;
             set => Model = value;
         }
 
-        public UpdateableRank(ScoreRank rank)
+        public UpdateableRank(ScoreRank? rank)
         {
             Rank = rank;
         }
 
-        protected override Drawable CreateDrawable(ScoreRank rank) => new DrawableRank(rank)
+        protected override void OnLoadFinished()
         {
-            Anchor = Anchor.Centre,
-            Origin = Anchor.Centre,
-        };
+            switch (RelativeSizeAxes)
+            {
+                case Axes.Y:
+                    Width = Rank.HasValue ? DrawHeight * 2 : 0;
+                    break;
+
+                case Axes.X:
+                    Height = Rank.HasValue ? DrawWidth / 2 : 0;
+                    break;
+            }
```

Comment:
This is not a good solution to the auto-sizing problem. I'd rather just revert to the previous version rather than allow this in.